app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -472,6 +496,49 @@ private void configureRightCard() {
         rightCardMapButton.setOnClickListener(v -> presenter.openCoordinateMap());
     }
 
+    @SuppressLint({"NewApi", "ClickableViewAccessibility"})
```

Comment:
why are we suppressing `NewApi` list warning? It could lead to crashes on lower Android versions. Please check this.